tensorflow/python/keras/benchmarks/benchmark_util.py

```diff
@@ -108,19 +110,26 @@ def measure_performance(model_fn,
   avg_epoch_time_list, wall_time_list, exp_per_sec_list = [], [], []
   total_num_examples = epochs * num_examples
 
+  strategy = distribution_util.get_distribution_strategy(
```

Comment:
Currently, the distribution_strategy is made to be off by default. But it will go through all the checking of dist-strat. This might be not some overhead for the current benchmark tests with no dist-strat.

I think either we preprocess strategy only when the arg `distribution_strategy` is not off, or we make some of our benchmark tests use dist-strat, so it's fine to leave the checking in the main branch. WDYT?

Reply:
Probably it's also good to have one or two benchmark tests using dist-strat with gpus. The gpu number is not necessary to be large, like one or two, as our model is not that large.

Some references: https://github.com/tensorflow/models/blob/0069c9328721f4f0aecb4de16d276d320f21876b/official/benchmark/keras_cifar_benchmark.py#L166

Reply:
> I think either we preprocess strategy only when the arg `distribution_strategy` is not off, or we make some of our benchmark tests use dist-strat, so it's fine to leave the checking in the main branch. WDYT?

I think we can set default value of `distribution_strategy` as `mirrored` and if there no gpus, it will use[ CPU instead](https://github.com/tensorflow/tensorflow/pull/41471/files#diff-9fb2f51ac75dc0b35e5e0a75bdaf9516R126), . So, it should be fine.

Reply:
> Probably it's also good to have one or two benchmark tests using dist-strat with gpus. The gpu number is not necessary to be large, like one or two, as our model is not that large.

Yeah, we can change the current benchmark tests later and use gpu.

Reply:
For now, as there is no benchmark test using dist-strat, it's okay to set it as off. Now num_gpus is also set as 0, so let's keep it as off.

If in the following PR we do have benchmarks running with dist-strat, it's okay to have dist-strat checking in the main branch. Let's make sure this will happen in next PRs then.

Reply:
Yes, we will add/change examples that will use dist-strat. Also, I checked code and if it is `off`, it returns None (and return dummy context), so it will not go through all checks in `distribution_util`.

Reply:
SGTM. Let's target dist-strat in next PR then.